build.sbt

```diff
@@ -24,18 +24,9 @@ lazy val jenaV = "5.3.0"
 lazy val rdf4jV = "5.1.3"
 lazy val titaniumApiV = "1.0.0"
 lazy val titaniumNqV = "1.0.2"
-// !! When updating ScalaPB also change the version of the plugin in plugins.sbt
-lazy val scalapbV = "0.11.17"
```

Comment:
Please look at /project. You need to remove the plugin, update build.sbt, and remove the metaprogramming stuff.